src/Views/Search/Search.js

```diff
@@ -1,11 +1,66 @@
-import React from 'react'
-
-const Search  = () => {
-    return ( 
-        <div>
-            Search
-        </div>
-      );
-}
- 
-export default Search;
\ No newline at end of file
+import { useState } from "react";
+import Book from "./Book";
+
+const Search = () => {
+  const [searchTitle, setSearchTitle] = useState("");
+  const [searchLocation, setSearchLocation] = useState("");
+  const [books, setBooks] = useState([]);
+  const [status, setStatus] = useState("Let's start searching!");
+
+  const handleInputChange = (e) => {
+    if (e.target.id === "searchTitle") return setSearchTitle(e.target.value);
+    if (e.target.id === "searchLocation")
+      return setSearchLocation(e.target.value);
+  };
```

Comment:
It looks quite strange for me. I would use separate function for every field. Can you elaborate why do it like that?

Reply:
Well, in this way I need less code. I saw this pattern in my course.

Reply:
fine by me :)